Makefile

```diff
@@ -11,13 +11,15 @@ build:
 	# size of binary.
 	go build -ldflags "-s" -o paralus .
 
-.PHONY: build-proto
-build-proto:
-	buf build
+.PHONY: clean-proto
+clean-proto:
+	rm -rf ./**/gen
+	find . -name "*.pb*" -type f -delete
 
-.PHONY: gen-proto
-gen-proto:
+.PHONY: build-proto
+build-proto: clean-proto
 	buf generate
+	buf build
```

Comment:
first would be buf build to ensure proto's are compiling and then buf generate

Reply:
Woops, fixed.